src/utils/cellSets/getContainingCellSetsProperties.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Returns the properties of the cellSets that contains the cellId
+ * (by the order specified in hierarchy children)
+ * @param {int} cellId The id of the cell we are searching for.
+ * @param {array} cellSetClassKeys The cellSetClass we are searching inside of
+ *  (e.g.: sample, louvain, scratchpad, etc..).
+ * @param {object} cellSets  cellSets for the experiment.
+ */
```

Comment:
Great job in adding the function documentation! Can I ask you to complete it by adding a `* @return {[type]}      [description]`?

Reply:
Copilot actually generated the comment btw :D